packages/@aws-cdk/aws-ec2/test/test.vpc-peering.ts

```diff
@@ -0,0 +1,231 @@
+import { expect, haveResource, haveResourceLike } from "@aws-cdk/assert";
```

Comment:
Thanks for the tests! To be completely sure, can you also add an integration test?